schema/validator.go

```diff
@@ -28,6 +30,12 @@ import (
 // and implements validation against a JSON schema.
 type Validator string
 
+type validateDescendantsFunc func(r io.Reader) error
+
+var mapValidateDescendants = map[Validator]validateDescendantsFunc{
+	MediaTypeManifest: validateManifestDescendants,
```

Comment:
> Please stop fomenting disagreement where there is none.

The maintainer position has been clear on this [since the 2016-10-12 meeting][1].  I don't think it was particular clear when I made these comments on 2016-10-05.

[1]: http://ircbot.wl.linuxfoundation.org/meetings/opencontainers/2016/opencontainers.2016-10-12-21.01.log.html#l-72